File: official/docs/csharp/v4/carrier-accounts/delete.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            EasyPost.Models.API.CarrierAccount carrierAccount = await client.CarrierAccount.Retrieve("ca_...");

            bool success = await carrierAccount.Delete();

            Console.WriteLine(success.ToString());
        }
    }
}

File: official/docs/golang/v4/carrier-accounts/delete.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v4"
)

func delete() {
	client := easypost.New("EASYPOST_API_KEY")

	err := client.DeleteCarrierAccount("ca_...")

	fmt.Println(err)
}
